mm/2s2h/BenPort.cpp

```diff
@@ -148,6 +149,7 @@ OTRGlobals::OTRGlobals() {
     // tell LUS to reserve 3 SoH specific threads (Game, Audio, Save)
     context =
         Ship::Context::CreateInstance("2 Ship 2 Harkinian", appShortName, "2ship2harkinian.json", archiveFiles, {}, 3);
+    prevAltAssets = CVarGetInteger("gAltAssets", 0);
```

Comment:
Looks like we are missing the complimentary initial set asset mode on the resource manager that you added on the SoH side.

```suggestion
    prevAltAssets = CVarGetInteger("gAltAssets", 0);
    context->GetResourceManager()->SetAltAssetsEnabled(prevAltAssets);
```